src/nuketesting/_cli/run_pytest_bootstrapped.py

```diff
@@ -0,0 +1,69 @@
+"""Script that gets executed by nuke to run tests.
+
+This script will be run through the `Runner` class. It adds pytest to the available packages and executes
+pytest with all provided arguments.
+"""
+# ! It is important here not to import anything not available in default Nuke,
+# ! as this is not bootstrapped yet.
+
+from __future__ import annotations
+
+import argparse
+import logging
+import sys
+from pathlib import Path
+from typing import NoReturn
+
+logger = logging.getLogger(__name__)
+
+
+class BootstrapException(Exception):
+    """Exception to raise during bootstrap."""
+
+
+def _run_tests(packages_directory: str, test_directory: str, pytest_arguments: list[str]) -> NoReturn:
+    """Run pytest with the provided arguments.
+
+    Note:
+        This function will exit the calling instance.
+
+    Args:
+        pytest_arguments:
+    """
+    for path in packages_directory.split(":"):
+        if not Path(path).is_dir():
+            msg = f"Package directory does not exist: '{path}'."
+            raise BootstrapException(msg)
+        sys.path.append(path)
+
+    import pytest
+
+    logging.info("Inserted packages for the NukeTestRunner successfully. Starting tests...")
```

Comment:
Actually this is a typo here, its using logging, it should be logger: as this is a getLogger call. This will either create a logger or use the one created: perfect for a usecase in a studio where a logger is already set-up.

Reply:
Yeah but if no logger is set up, the message will not print to standard out. It needs to be configured to work.